MAINTAINERS.md

```diff
@@ -4,7 +4,13 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 
 ## Current Maintainers
 
-| Maintainer | GitHub ID                                       | Affiliation |
-| ---------- | ----------------------------------------------- | ----------- |
-| David Cui  | [davidcui1225](https://github.com/davidcui1225) | Amazon      |
-| Joshua Li  | [joshuali925](https://github.com/joshuali925)   | Amazon      |
+| Maintainer        | GitHub ID                                         | Affiliation |
+| ----------------- | ------------------------------------------------- | ----------- |
```

Comment:
we might need a vote to get @ps48 or @joshuali925 onto this list

Reply:
Let's make sure the PR code errors in codeowners file are expected. 

```


    Unknown owner on line 2: make sure @pjfitzgibbons exists and has write access to the repository

    *   @pjfitzgibbons @ps48 @kavithacm @derek-ho @joshuali925 @dai-chen @YANG-DB @rupal-bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az

    Unknown owner on line 2: make sure @derek-ho exists and has write access to the repository

    …jfitzgibbons @ps48 @kavithacm @derek-ho @joshuali925 @dai-chen @YANG-DB @rupal-bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az

    Unknown owner on line 2: make sure @dai-chen exists and has write access to the repository

    …ithacm @derek-ho @joshuali925 @dai-chen @YANG-DB @rupal-bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az

    Unknown owner on line 2: make sure @swiddis exists and has write access to the repository

    …bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az

    Unknown owner on line 2: make sure @penghuo exists and has write access to the repository

    …eieric @vamsi-amazon @swiddis @penghuo @seankao-az

    Unknown owner on line 2: make sure @seankao-az exists and has write access to the repository

    …amsi-amazon @swiddis @penghuo @seankao-az

```